### DISCUSSION TITLE
Performance difference between 2022.628.0 and 2022.605.1

### DISCUSSION THREAD

**Discussion by Taitranz** [ppy/osu#18934] [Q&A]
After further testing of 2022.628.0 and 2022.605.1. The performance difference is still very noticeable. 

2022.628.0 was an improvement over 2022.612.0
Build 2022.605.1 outperforms 2022.628.0 in both performance of framerate and perceivable latency.

Is there anything that can be done about this?

**Comment by peppy** [comment]
Take a video from both builds with frame graph visible (Ctrl+F11 twice), and explain how you are perceiving the latency.

**Reply by Taitranz** [reply]
Yes I will do that please give me abit

**Reply by Taitranz** [reply]
Build 2022.605.1 : https://www.youtube.com/watch?v=byL_EHu-nLA
Build 2022.628.0 : https://www.youtube.com/watch?v=cIvfHuORzOQ

The latency that I am perceiving is input delay.
My best example is when I move the pen or mouse, the delay between what is occurring in real life and when it is moving on the screen is noticeable for me, especially when the cursor is moving at higher speeds. I believe for most user using 144hz it may not be noticeable but for users using higher refresh rates such as 240hz and above, it is noticeable. 

Thank you for reading and sorry for the poor wording.

**Reply by peppy** [reply]
Thanks. Everything looks pretty good as far as the numbers show on your videos.

I will prepare a test build in the coming days to confirm a theory, but can you check whether your phone can take video using slow motion (240fps or higher)? This would be helpful in diagnosing further.

**Reply by Taitranz** [reply]
Yes I will try

**Reply by peppy** [reply]
Before we go further, can you confirm that the **windows mouse cursor** feels as good as (or better?) than 2022.605.1?

**Reply by Taitranz** [reply]
I've been using the latest build for the last 2 days I can say that the latest build feels as good as 2022.605.1. I am unable to record my screen with my phone (unless you want the automated gameplay of Osu). There seems to be a performance decrease when using automation on 2022.605.1. I can provide a automated playthrough if that would help.

Thank you

**Reply by peppy** [reply]
Great to hear.

Are you seeing the "performance decrease" in autoplay in terms of total draw frames, or actually dropping so low that it's visual on your screen?

**Reply by Taitranz** [reply]
Hi, sorry for the late response.  2022.605.1 has a decreased in draw frames when using autoplay but the visual looks the same.

**Reply by peppy** [reply]
That sounds as expected then, thanks.

**Comment by Taitranz** [comment]
Video of difference in build with HUD off 

Build 2022.605.1 : https://www.youtube.com/watch?v=mcWzTcQuu4Q
Build 2022.628.0 : https://www.youtube.com/watch?v=SCDie_57j70

**Reply by peppy** [reply]
Can you please test with the HUD on? Since I know for a fact that HUD off will have a minor discrepancy.

**Reply by Taitranz** [reply]
I will post one with graphs soon and hud on
